### PR TITLE
Form validation and tests

### LINKED ISSUE #9: SignUp Form validation and functionality
# REQUIREMENTS
-Sign Up button must be disabled until the user provides the valid information.
-Error messages must be displayed if the user does not provide a correct value for the input.
-The border color of each TextInput must change once the value provided is correct.
# DESIGN

# Acceptance Criteria
List all required acceptance criteria
- [ ] Criteria 1
- [ ] Criteria 2

### PR DESCRIPTION
**Pull request by Joaquin-Niembro**
Co-authored-by: Juan Carbajal <jjcr0397@gmail.com>
Co-authored-by: Joaquin Niembro Bueno <joaquinniembro@gmail.com>

# Description
-We implemented form validation as well as style changes as you valid the inputs.
-Validation triggers as you type.
-Submit Function cannot be access if it is not validated.
-When validated submit sends a console log with the values. (We have not implemented firebase/auth yet but will be added soon).
 
 Examples:
-we have added unit and snapshot testing for all of our components using jest and Enzyme.

 # Testing
-Git Clone od the repository
-cd ./ReservacionDeVuelos
-git switch feature#9/SignUpFormValidation
-npm install
-npm run start
-npm run android/ios
 
  
  ## Screenshots 
vaidat
blue
de
s
q
testcoverage

### CODE REVIEW

**Review by sebasdrk17 — DISMISSED** [review]
Looks great!

**Comment by vmagana3** [comment]
Looks good, respect the user inputs states!

**Review by vmagana3 — DISMISSED** [review]
Looks good, it respect the user inputs states!

**Review by JordiGN — APPROVED** [review]
Great, and also use of tests, that's impressive, amazing job!

**Review by tovl1 — APPROVED** [review]
Good job!

File: ReservacionDeVuelos/src/utils/Functions/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './SignUpFunctions'
```
Comment (LeyaimMagma) [inline ReservacionDeVuelos/src/utils/Functions/index.js:1]: Add a new space

**Review by LeyaimMagma — APPROVED** [review]

File: ReservacionDeVuelos/src/utils/Functions/index.js (hunk 1 above)
Comment (Joaquin-Niembro) [reply ReservacionDeVuelos/src/utils/Functions/index.js:1]: Yes we will!

**Pull request merged**